examples/model_assisted_labeling/image_mal.ipynb

```diff
@@ -23,13 +23,7 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    !pip install labelbox requests ndjson scikit-image PILLOW tensorflow opencv-python
</pre><p>Maybe add the <code>-q</code> flag  here too</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/375/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>